domains/doriancappe.json

```diff
@@ -0,0 +1,11 @@
+{
+    "description": "My Portfolio",
+    "repo": "https://github.com/fiduide/portfolio/",
+    "owner": {
+        "username": "fiduide",
+        "email": "xxreflexdu92xx@hotmail.fr",
+    },
+    "record": {
+        "CNAME": "fiduide.github.io"
```

Comment:
404?

Reply:
https://fiduide.github.io/portfolio/

Reply:
@fiduide Sorry the site has to be on the main page: https://fiduide.github.io you can do this by changing your portfolio repo name to `fiduide.github.io`. Also, the site needs to be in English.